.github/workflows/react.yaml

```
@@ -23,4 +23,5 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
     - run: npm ci
+    - run: npm run test
```

@shubhiroy 
```suggestion
    - run: npm run test
```
Remove this line